examples/apps/ai-collab/next.config.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
```

Comment:
Seeing error: /home/tong/workspace/FluidFramework/node_modules/.pnpm/next@14.2.8_@babel+core@7.25.7_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/server/config.js:787
            throw new Error(`Configuring Next.js via '${(0, _path.basename)(nonJsPath)}' is not supported. Please replace the file with 'next.config.js' or 'next.config.mjs'.`);

including next.config.ts in tsconfig.json...see how it goes

Reply:
rename to .cjs works. Thx!